docs/source/deployment/databricks.md

```diff
@@ -223,3 +223,68 @@ Out[12]: {}
 Your complete notebook should look similar to this (the results are hidden):
 
 ![](../meta/images/databricks_notebook_example.png)
+
+
+### 9. Using IPython Extension 
+
+You can interact with Kedro in Databricks through the Kedro [IPython extension](https://ipython.readthedocs.io/en/stable/config/extensions/index.html), `kedro.extras.extensions.ipython`.
+
+The Kedro IPython extension launches a [Kedro session](../kedro_project_setup/session.md) and makes available the useful Kedro variables `catalog`, `context`, `pipelines` and `session`. It also provides the `%reload_kedro`  [line magic](https://ipython.readthedocs.io/en/stable/interactive/magics.html) that reloads these variables (for example, if you need to update `catalog` following changes to your Data Catalog).
+
+The IPython extension can be used in a Databricks notebook in a similar way to how it is used in [Jupyter notebooks](https://kedro.readthedocs.io/en/0.18.0/tools_integration/ipython.html)
+
+There's one additional installation you need to do in the Databricks notebook to make use of the IPython extension. After you load the IPython extension using the below command:
+
+```ipython
+In [1]: %load_ext kedro.extras.extensions.ipython
+```
+
+you must explicitly upgrade your pip version by doing the below:
+
+```bash
+%pip install -U pip
+```
+
+after this, you can reload kedro by running the line magic command  `%reload_kedro <path_to_project_root>`.
+
+### 10. Running Kedro-viz on Databricks 
```

Comment:
```suggestion
### 10. Running Kedro-Viz on Databricks 
```

Reply:
Remember to capitalise the `v` in `Kedro-Viz`, [according to the style guide](https://kedro.readthedocs.io/en/stable/contribution/documentation_contributor_guidelines.html#kedro-lexicon). I spotted this one a few times but I don't want to leave comments for it.